units/system-update.target

```diff
@@ -12,7 +12,6 @@ Description=Offline System Update
 Documentation=man:systemd.offline-updates(7)
 Documentation=man:systemd.special(7) man:systemd-system-update-generator(8)
 Requires=sysinit.target
-After=sysinit.target
+After=sysinit.target system-update.target
```

Comment:
`system-update-pre.target` doesn't cause any additional synchronization (it is only After=sysinit.target, but `systemd-update.target` is the same). It can be made passive, but this doesn't really change much, except removing one job from the queue (which is a good thing though).

Reply:
Erm, typo that should be "After=systemd-update-pre.target" not "After=systemd-update.target"

Reply:
Ooops. Fix in https://github.com/systemd/systemd/pull/9357.